src/components/custom/_signup-new.js

```diff
@@ -194,6 +194,17 @@ const StyledBinaryLogo = styled.img`
         width: 40px;
     }
 `
+const FooterNote = styled(Text)`
```

Comment:
`Text` component is deprecated please use `Header` instead and use `as` prop to create the element based on specification.